test/test_turtle_w3c.py

```diff
@@ -28,6 +28,7 @@ def turtle(test: RDFTest):
 
         if test.result:  # eval test
             res = Graph()
+            assert not isinstance(test.result, tuple)
```

Comment:
This assert is here because `test.result` can be a tuple or Identifier, and if it is a tuple then it cannot be use as a parse source.